src/rpc/mod.rs

```diff
@@ -485,6 +486,13 @@ where
                     svc_builder,
                     keystore,
                 } = per_conn.clone();
+                let http_middleware = tower::ServiceBuilder::new()
+                    // This does not compile yet, a fix has been made in https://github.com/paritytech/jsonrpsee/pull/1475
+                    // .layer(CompressionLayer::new())
```

Comment:
I can uncomment this line in a subsequent PR once a new version of `jsonrpsee` is released.